active/fellowship.md

```diff
@@ -2,30 +2,67 @@
 
 ## Scope of responsibilities
 
-The Fellowship Working Group manages the operation of the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). They select candidates to fill the fellowship, and act as line managers during the operation of the program.
+The Fellowship Working Group manages the [Django Fellowship Program](https://www.djangoproject.com/fundraising/#fellowship-program). This includes:
+
+- Interviewing candidates for open Fellowship slots and making hiring recommendations to the Board. This includes developing and maintaining interview and selection scripts, exercises, and rubrics.
```

Comment:
> This includes developing and maintaining interview and selection scripts, exercises, and rubrics.

We didn't interview previously, want to highlight this before we document that interviews are required.

Reply:
Agreed, we have done this purely on written applications and recommendations before, and even that takes a decent chunk of time; adding interviews would make it be a huge cost for the WG. It would be nice to leave the _option_, but let's not phrase it like it's expected?